src/webapp/components/copy-in-user-dialog/CopyInUserDialog.tsx

```diff
@@ -79,53 +64,65 @@ export const CopyInUserDialog: React.FC<CopyInUserDialogProps> = props => {
             onSave={onDialogSave}
         >
             <Container>
-                <Toggle
-                    label={i18n.t("Bulk update strategy: {{strategy}}", {
-                        strategy: strategyLabel,
-                        nsSeparator: false,
-                    })}
-                    style={{ width: 280, float: "inline-end", marginBlockStart: 20, marginInlineStart: 15 }}
-                    toggled={isReplaceStrategy}
-                    onToggle={onToggleStrategy}
-                />
-                <MultiSelector
-                    d2={d2}
-                    height={300}
-                    onChange={setSelectedUsersIds}
-                    options={getOptions()}
-                    ordered={false}
-                    searchFilterLabel={i18n.t("Search by name")}
-                    classes={{
-                        searchField: useStyles().searchFieldOverride,
-                    }}
```

Comment:
> Thanks, @delemaf. It looks good, but I think there was a confusion about the library. Our library is [d2-ui-components](https://github.com/EyeSeeTea/d2-ui-components?tab=readme-ov-file#multiple-selector), and we have a `<MultipleSelector />` component.
> 
> Do you think you can take a look at it and, if possible, use that component instead of the one from d2-ui? You can see an [example in bulk load](https://github.com/EyeSeeTea/Bulk-Load/blob/dd92964484c77a9a0ddef5f268183fd02f7136c1/src/webapp/components/settings/DataFormTemplateAssignDialog.tsx#L104). Let me know if you have any questions.

It was initially a `MultiSelector` from d2-ui-components but I saw a more recent code with d2-ui that looked cleaner, that's why I asked question to @MiquelAdell [here](https://github.com/EyeSeeTea/user-extended-app/pull/287#issuecomment-2139088849)

Reply:
Got it. My bad I thought your initial question was for migrate from `dhis-ui` to `d2-ui-components` (our library). I'm not sure about this. I think is safe to keep using our library since we have control to do any modification, but I will leave this decision up to @MiquelAdell.